extensions/wikia/AdEngine/js/lookup/prebid/adapters/fastlane.js

```diff
@@ -0,0 +1,162 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.fastlane', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.context.slotsContext',
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.geo',
+	'wikia.instantGlobals',
+	'wikia.log'
+], function (adContext, slotsContext, adLogicZoneParams, geo, instantGlobals, log) {
+	'use strict';
+
+	var bidderName = 'fastlane',
+		aliases = {
+			'rubicon': [bidderName]
+		},
+		logGroup = 'ext.wikia.adEngine.lookup.prebid.adapters.fastlane',
+		rubiconAccountId = 7450,
+		slots = {
+			oasis: {
+				TOP_LEADERBOARD: {
+					sizes: [[728, 90], [970, 250]],
+					targeting: {loc: 'top'},
+					siteId: 41686,
+					zoneId: 175094
+				},
+				TOP_RIGHT_BOXAD: {
+					sizes: [[300, 250], [300, 600], [300, 1050]],
+					targeting: {loc: 'top'},
+					siteId: 41686,
+					zoneId: 175094
+				},
+				LEFT_SKYSCRAPER_2: {
+					sizes: [[120, 600], [160, 600], [300, 250], [300, 600], [300, 1050]],
+					targeting: {loc: 'middle'},
+					siteId: 41686,
+					zoneId: 194452
+				},
+				LEFT_SKYSCRAPER_3: {
+					sizes: [[120, 600], [160, 600], [300, 250], [300, 600]],
+					targeting: {loc: 'footer'},
+					siteId: 41686,
+					zoneId: 194452
+				},
+				INCONTENT_BOXAD_1: {
+					sizes: [[120, 600], [160, 600], [300, 250], [300, 600]],
+					targeting: {loc: 'hivi'},
+					siteId: 83830,
+					zoneId: 395614
+				},
+				BOTTOM_LEADERBOARD: {
+					sizes: [[728, 90], [970, 250]],
+					targeting: {loc: 'footer'},
+					siteId: 41686,
+					zoneId: 194452
+				},
+				PREFOOTER_LEFT_BOXAD: {
+					sizes: [[300, 250], [336, 280]],
```

Comment:
Please limit slot sizes to avoid customs. For reference see appnexus config: https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js#L13-L93